src/main/java/org/junitpioneer/vintage/Test.java

```diff
@@ -44,6 +44,9 @@
 @org.junit.jupiter.api.Test
 public @interface Test {
 
+	/**
+	 * Dummy default class for the `expected` parameter.
```

Comment:
Markdown doesn't work here. 😉

```suggestion
	 * Dummy default class for the <code>expected</code> parameter.
```